jconstraints-metasolver/src/test/java/tools/aqua/jconstraints/solvers/AbstractTest.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2015 United States Government, as represented by the Administrator
+ *                of the National Aeronautics and Space Administration. All Rights Reserved.
+ *           2017-2021 The jConstraints Authors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package tools.aqua.jconstraints.solvers;
+
+import org.testng.SkipException;
+import org.testng.annotations.BeforeMethod;
+
+public class AbstractTest {
+
+  @BeforeMethod
+  public void initialize() {
+    if (System.getProperty("os.name").toLowerCase().contains("win")) {
+      throw new SkipException("No native CVC4 support");
```

Comment:
This must be migrated to JUnit5 API
```suggestion
import static org.junit.jupiter.api.Assumptions.assumeFalse;

import org.junit.jupiter.api.BeforeAll;
import org.opentest4j.TestAbortedException;

public class AbstractTest {

  @BeforeAll
  public static void initialize() {
    assumeFalse(System.getProperty("os.name").toLowerCase().contains("win"));
```